netlify/cli
# Revert "Remove shrink-wrap pruning during pack step"

**Summary**
This reverts commit c28bd8189b5a9e602a249e76c857c2d6af858ce4.

Dev dependencies listed in `npm-shrinkwrap.json` are getting installed when the CLI is installed globally.

This PR brings back the fix we had in place so only production dependencies are included before publishing.

**Test plan**
1. Run `npm pack`
2. Verify that `npm-shrinkwrap.json` in generated tarball doesn't include dev dependencies, such as ava.

**Description for the changelog**
Prune npm-shrink-wrap during pack step

**A picture of a cute animal (not mandatory but encouraged)**
![corgi jumping on the beach](https://user-images.githubusercontent.com/83225/59864857-90e8af80-933c-11e9-8d27-d6d845e66615.jpg)
